phpunit.xml

```diff
@@ -10,7 +10,7 @@
          beStrictAboutTodoAnnotatedTests="true"
          failOnRisky="true"
          failOnWarning="true"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd"
```

Comment:
This is an unrelated change - if you're experiencing some error due to this, please open a separate PR with an explanation, thanks.

Reply:
there's no error but this project uses [phpunit 9.5](https://github.com/phpstan/phpstan-doctrine/blob/master/composer.json#L30). I can leave it for a separate PR since it's unrelated if you think that's best, I just thought it's a minor thing

Reply:
Yeah, it's minor, but I'm interested in how did you notice this - does PHPUnit print a warning or something?

Reply:
no warning, I was just curious about the config :)

Reply:
Yeah, so please remove this change and feel free to submit it separately, but I don't thinkt it's necessary to touch it :)